CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Change Log
 ==========
 
+**Unreleased**
+
+* :meth:`Submission.mod.set_original_content` support 
+  marking a submission as "original content."
```

Comment:
Can this be changed to the following:

    * :meth:`.set_original_content` supports
      marking a submission as original content.

I changed the `:meth:` reference so Sphinx (the documentation generator) can find the proper target, I changed "support" to "supports" as a grammar fix, and I removed the quotation marks from "original content" because they seemed unnecessary.